internal/quickstart/choose_sdk.go

```diff
@@ -40,7 +40,6 @@ func NewChooseSDKModel(selectedIndex int) tea.Model {
 	l.SetShowPagination(true)
 	l.SetShowStatusBar(false)
 	l.SetFilteringEnabled(false) // TODO: try to get filtering working
-	l.Paginator.PerPage = 5
```

Comment:
We're not allowed to manually set this unfortunately. The property gets computed automatically by list.updatePagination(). So setting it manually messes up the computation. I found a way to show exactly 5 items or less on every page though by setting the hight on line 34 to 9.